[BUG] orc_test is failing on Spark 3.2+

After #6058 orc_test is failing on Spark 3.2+ with:
```
../../src/main/python/orc_test.py:178: in <module>
    if not is_before_spark_320() and not is_spark_321cdh():
E   NameError: name 'is_spark_321cdh' is not defined
```

--- a/integration_tests/src/main/python/orc_test.py
+++ b/integration_tests/src/main/python/orc_test.py
@@ -18,7 +18,7 @@ from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_co
 from data_gen import *
 from marks import *
 from pyspark.sql.types import *
-from spark_session import with_cpu_session, is_before_spark_320, is_before_spark_330
+from spark_session import with_cpu_session, is_before_spark_320, is_before_spark_330, is_spark_321cdh
 from parquet_test import _nested_pruning_schemas
 from conftest import is_databricks_runtime
 

